tests/e2e/actions.go

```diff
@@ -35,18 +35,18 @@ func (tr TestRun) sendTokens(
 	action SendTokensAction,
 	verbose bool,
 ) {
-	binaryName := tr.chainConfigs[action.chain].binaryName
+	BinaryName := tr.chainConfigs[action.Chain].BinaryName
```

Comment:
Still a local var right?
```suggestion
	binaryName := tr.chainConfigs[action.Chain].BinaryName
```